### DISCUSSION TITLE
ERROR!

### DISCUSSION THREAD

**Discussion by beatriz-mano** [biigle/core#776] [Q&A]
Hi, When I try to open my pictures appears this 'Failed to load image 4373872!'. What happens? Can you help me?

**Comment by dlangenk** [comment]
Hi Beatriz,

the problem is that these are still tiff images, although they are named png (see image). You have to convert these and upload them again, as tiff images cannot be opened by a browser. This is no limitation of Biigle but of the Browser itself.

<img width="320" alt="image" src="https://github.com/biigle/core/assets/4822085/cee43a06-2c1a-430e-859b-bb1234022de2">
